atlasdb-jepsen-tests/src/jepsen/atlasdb/timestamp.clj

```diff
@@ -58,18 +58,18 @@
 ;; Defining the Jepsen test
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 (defn timestamp-test
-  []
+  [nem]
   (assoc tests/noop-test
     :os debian/os
     :client (create-client nil)
-    :nemesis (nemesis/partition-random-halves)
+    :nemesis nem
     :generator (->> read-operation
-                    (gen/stagger 0.05)
-                    (gen/nemesis
-                    (gen/seq (cycle [(gen/sleep 5)
-                                     {:type :info, :f :start}
-                                     (gen/sleep 85)
-                                     {:type :info, :f :stop}])))
-                    (gen/time-limit 360))
+                 (gen/stagger 0.05)
+                 (gen/nemesis
+                   (gen/seq (cycle [(gen/sleep 5)
+                                    {:type :info, :f :start}
+                                    (gen/sleep 15)
+                                    {:type :info, :f :stop}])))
+                 (gen/time-limit 180))
```

Comment:
But it's gone from running a partition_random_halves for 360 seconds, to running partition_random_halves for 180 and then crash for 180, right? I'd keep it at 360, I don't think we need to worry about the run time so much.